### ISSUE TITLE
Define a capabilities mechanism

### ISSUE THREAD

**Issue by pthatcherg** [webscreens/openscreenprotocol#123]
We need a way to know which parts of the protocol an endpoint supports: does it allow Presentations? Remote Playback?  Streaming?  And if we make a new version of a protocol, does it support that?  We need a mechanism for this.

**Comment by mfoltzgoogle** [comment]
Should ensure that any capabilities disclosed before authentication are discussed in the Security & Privacy section, and the possible implications thereto.

**Comment by rektide** [comment]
My ask for audio/video capabilities appears to be the primary thing implemented in the capabilities PR, https://github.com/webscreens/openscreenprotocol/pull/140/files . 👍

**Comment by mfoltzgoogle** [comment]
Closing this as we've landed spec changes:
- For agents to exchange capabilities
- A list of capabilities defined by the spec
- A way for agents to advertise extended (custom) capabilities
- A registry of extended capabilities
- Noting that capabilities are advertised to any network client and are public information.

The remaining open issue around capabilities is #200, but that is specific to receives-audio and receives-video.   Closing this issue for now based on the current state of the spec.
